_scripts/build.js

```diff
@@ -7,13 +7,22 @@ const { name, productName } = require('../package.json')
 const args = process.argv
 
 let targets
-var platform = os.platform()
+const platform = os.platform()
+const cpus = os.cpus()
 
-if (platform == 'darwin') {
-  targets = Platform.MAC.createTarget()
-} else if (platform == 'win32') {
+if (platform === 'darwin') {
+  let arch = Arch.x64
+
+// Macbook Air 2020 with M1 = 'Apple M1'
+  // Macbook Pro 2021 with M1 Pro = 'Apple M1 Pro'
+  if (cpus[0].model.startsWith('Apple')) {
+    arch = Arch.arm64
+  }
+
+  targets = Platform.MAC.createTarget(['dmg'], arch)
```

Comment:
I don't have a MAC and I'm not entirely sure how to test but I think 
this line might need to be changed to this? 
```suggestion
  targets = Platform.MAC.createTarget(['dmg', 'zip'], arch)
```

Reply:
I don't think FT has mac package with `zip` format right now
- https://github.com/FreeTubeApp/FreeTube/actions/runs/2296005800
- https://github.com/FreeTubeApp/FreeTube/releases/tag/v0.16.0-beta